packages/accounts/src/light-account/clients/multiOwnerLightAccount.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  createSmartAccountClient,
+  type SmartAccountClient,
+  type SmartAccountClientActions,
+  type SmartAccountClientConfig,
+  type SmartAccountSigner,
+  type SmartContractAccount,
+} from "@alchemy/aa-core";
+import { type Chain, type CustomTransport, type Transport } from "viem";
+import {
+  createMultiOwnerLightAccount,
+  type CreateMultiOwnerLightAccountParams,
+  type MultiOwnerLightAccount,
+} from "../accounts/multiOwner.js";
+import {
+  multiOwnerLightAccountClientActions,
+  type MultiOwnerLightAccountClientActions,
+} from "../decorators/multiOwnerLightAccount.js";
+
+export type CreateMultiOwnerLightAccountClientParams<
+  TTransport extends Transport = Transport,
+  TChain extends Chain | undefined = Chain | undefined,
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = {
+  transport: CreateMultiOwnerLightAccountParams<
+    TTransport,
+    TSigner
+  >["transport"];
+  chain: CreateMultiOwnerLightAccountParams<TTransport, TSigner>["chain"];
+  account: Omit<
+    CreateMultiOwnerLightAccountParams<TTransport, TSigner>,
+    "transport" | "chain"
+  >;
+} & Omit<
+  SmartAccountClientConfig<TTransport, TChain>,
+  "transport" | "account" | "chain"
+>;
+
+export function createMultiOwnerLightAccountClient<
+  TChain extends Chain | undefined = Chain | undefined,
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+>(
+  args: CreateMultiOwnerLightAccountClientParams<Transport, TChain, TSigner>
+): Promise<
+  SmartAccountClient<
+    CustomTransport,
+    Chain,
+    MultiOwnerLightAccount<TSigner>,
+    SmartAccountClientActions<Chain, SmartContractAccount> &
```

Comment:
nit: you shouldn't need to specify `SmartAccountClientActions<Chain, SmartContractAccount>` anymore after the multisig PR. This will always be included in the actions list